src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNScoreVariantsIntegrationTest.java

```diff
@@ -99,6 +99,19 @@ public void testSmallBatchInference()throws IOException {
         spec.executeTest("testInference", this);
     }
 
+    @Test(groups = {"python"})
+    public void testOnContigEdge() throws IOException{
+        final String edgeVcf = toolsTestDir + "walkers/VQSR/variantNearContigEdge.vcf";
+        final ArgumentsBuilder argsBuilder = new ArgumentsBuilder();
+        argsBuilder.addArgument(StandardArgumentDefinitions.VARIANT_LONG_NAME, edgeVcf)
+                .addArgument(StandardArgumentDefinitions.REFERENCE_LONG_NAME, hg19MiniReference)
+                .addArgument("architecture", architecture1D)
```

Comment:
@lucidtronix I just noticed this test is using 1d, which doesn't exercise the changed code path.

Reply:
I think the users who encountered this error were running the 1D model, but either way both models will call the reference_string_to_tensor function in inference.py where the null character check was added.